extensions/amp-animation/0.1/web-animations.js

```diff
@@ -840,14 +847,19 @@ class CssContextImpl {
    * @param {!Window} win
    * @param {!Document|!ShadowRoot} rootNode
    * @param {string} baseUrl
+   * @param {!./web-animation-types.WebAnimationBuilderOptionsDef} options
    */
-  constructor(win, rootNode, baseUrl) {
+  constructor(win, rootNode, baseUrl, options) {
     /** @const @private */
     this.win_ = win;
 
-    /** @const @private */
+    /** @const @private {!Node} */
     this.rootNode_ = rootNode;
 
+    const {scope = rootNode} = options;
+    /** @const @private {!Document|!ShadowRoot} */
```

Comment:
This is the other way around :). The `rootNode_` above is `Document|ShadowRoot`, and thus the `this.scope_` becomes `Node`.